.travis.yml

```diff
@@ -33,26 +34,41 @@ jobs:
     - stage: "dep check"
       script: make depcheck
       name: " Check if imports, Gopkg.toml, and Gopkg.lock are in sync"
-    - stage: "Test Jobs"                # naming the Tests stage
+
+    - stage: "Test Build"                # naming the Tests stage
       script: make 
       name: "builds cloud and edge components"            # names the first Tests stage job
+    - script: make edge_cross_build
+      name: "cross build edge"     # names the fifth Tests stage job
+    - script: make bluetoothdevice
+      name: "build bluetooth_device"
+    - script: make edge_small_build
+      name: "build small edge version"
+    - stage: "Test lint"
+      script: make cloud_lint
+      name: "cloud_lint"     # names the eighth Tests stage job
+    - script: make bluetoothdevice_lint
+      name: "bluetooth_device_lint"
+    - script: make keadm_lint
+      name: "keadm_lint"
     - script: make edge_verify
       name: "verify edge"     # names the second Tests stage job
-    - script: make edge_test
+    - stage: "unit tests"
+      script: make edge_test
       name: "unit test edge"     # names the third Tests stage job
     - script: make edge_integration_test
```

Comment:
we also have some integration tests in the cloud for device crd validation. Can we include those?
These tests need a K8S cluster and cloud component up and running.

Reply:
Yes we can include these tests. But I think there is an issue on k8s side. Compare to the message in the main makefile:
`This has been commented temporarily since there is an issue of CI using same master for all PRs, which is causing failures when run parallely`

Reply:
This is not blocking, I'd suggest we discuss it in the tracking issue and get this accepted.